src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -62,6 +62,7 @@
     protected EnumSet<Message.MentionType> allowedMentions;
     protected Set<String> mentionableUsers = new HashSet<>();
     protected Set<String> mentionableRoles = new HashSet<>();
+    protected long messageReference;
```

Comment:
Why not stick with `referencedMessage`?

Reply:
Nvm, I looked down, and the raw JSON is called `message_reference`, I guess that's why. But then why is everything else referencedMessage?